docs/graphql/manual/guides/deployment/digital-ocean-managed-kubernetes.rst

```diff
@@ -0,0 +1,234 @@
+Hasura GraphQL Engine on DigitalOcean Managed Kubernetes
+===============================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+DigitalOcean now offers a managed Kubernetes service, making it a perfect hosting provider to
+quickly setup and deploy Hasura GraphQL Engine without managing servers.
+
+Quickstart
+----------
+
+Following this tutorial means you will need the following installed on your machine:
+
+- ``doctl`` (https://github.com/digitalocean/doctl/blob/master/README.md)
+- ``kubectl`` (https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+Setting up a Managed Database
+----------
+
+Hasura needs a PostgreSQL database, to keep things easy, we can use a Digital Ocean Managed database.
+
+Login to your Digital Ocean console and create a new managed database, select the latest version
+of PostgreSQL and name your database ``hasura-demo``.
+
+.. thumbnail:: ../../../../img/graphql/manual/guides/digital-ocean-managed-kubernetes/create-managed-database.png
+
+While that is provisioning, we can move onto the next steps.
+
+Setting up a Managed Cluster
+----------
+
+The fastest way to setup a Kubernetes cluster on Digital Ocean is to use the `doctl` command line tool.
+Let's create a cluster and call it ``hasura-demo``.
+
+.. note::
+
+  Feel free to rename this to match your project name but the remainder of this tutorial will refer to
+  the cluster as ``hasura-demo`` so don't forget to change the name to match!
+
+The following command will create a cluster using the ``doctl`` command and spin up three worker nodes.
+
+.. code-block:: bash
+
+  doctl kubernetes cluster create hasura-demo
```

Comment:
@jasonmccallister I'm a bit stuck on this command. I get the following error:

```
unable to initialize DigitalOcean api client: access token is required. (hint: run 'doctl auth init')
```

Can we add something here what is the best way to initialise auth?